cmd/google.go

```diff
@@ -87,7 +89,7 @@ var (
 			logger.Log("msg", "starting terracognita", "version", Version)
 			err = provider.Import(ctx, googleP, hclW, stateW, f, logsOut)
 			if err != nil {
-				return fmt.Errorf("could not import from google: %+v", err)
+				return fmt.Errorf("could not import from google: %v", err)
```

Comment:
In the code below you're using `pkg/errors` for wrapping, so it could be also used there instead of `fmt`.